### PR TITLE
DOC: update the DataFrame.sub docstring

### PR DESCRIPTION
**Pull request by limitup**
Checklist for the pandas documentation sprint :

- [x] PR title is "DOC: update the <your-function-or-method> docstring"
- [x] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [x] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [x] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	Use only one blank line to separate sections or paragraphs
	Errors in parameters section
		Wrong parameters order. Actual: ('other', 'axis', 'level', 'fill_value'). Documented: ('other', 'axis', 'fill_value', 'level')
		Parameter "other" has no description
		Parameter "axis" description should finish with "."
		Parameter "fill_value" description should finish with "."
		Parameter "level" description should finish with "."
	Missing description for See Also "DataFrame.rsub" reference

```

Validation script returns docstring style errors.

### CODE REVIEW

File: pandas/core/ops.py [hunk 1]
```diff
@@ -370,16 +370,44 @@ def _get_op_name(op, special):
 e  NaN  2.0
 """
 
+_sub_example_FRAME = """
+>>> a = pd.DataFrame([2, 1, 1, np.nan], index=['a', 'b', 'c', 'd'],
+                      columns=['one'])
```
Comment (jorisvandenbossche) [inline pandas/core/ops.py]: You need to add `...` at the beginning of the line here.

The same for `b` below

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks for the PR! 

Can you ensure with `scripts/validate_docstring.py` that everything is OK? 
For me this gives errors for the examples.

**Comment by kurtiskerstein** [comment]
I accidentally closed this PR.  Reopening

**Comment by kurtiskerstein** [comment]
When I added the "..." for the wrapped lines there were no more validation errors.

**Comment by jorisvandenbossche** [comment]
@kurtiskerstein you seem to have included some unrelated commits. If you need any help with resolving this, let us know.

**Comment by kurtiskerstein** [comment]
Yeah.  I am going back through the  git documentation to figure it out.  I will let you know if I have further trouble.

File: pandas/core/ops.py [hunk 2]
```diff
@@ -354,7 +354,7 @@ def _get_op_name(op, special):
 d  NaN
 >>> b = pd.DataFrame(dict(one=[1, np.nan, 1, np.nan],
 ...                       two=[np.nan, 2, np.nan, 2]),
-...                  index=['a', 'b', 'd', 'e'])
+...                       index=['a', 'b', 'd', 'e'])
```
Comment (jorisvandenbossche) [inline pandas/core/ops.py]: the index is not part of the dict, so the indentation was actually correct

File: pandas/core/ops.py [hunk 3]
```diff
@@ -370,16 +370,44 @@ def _get_op_name(op, special):
...
+e  NaN  -2.0
+"""
+
 _op_descriptions = {
     # Arithmetic Operators
     'add': {'op': '+',
             'desc': 'Addition',
             'reverse': 'radd',
             'df_examples': _add_example_FRAME},
+    # Subtraction Operators
```
Comment (jorisvandenbossche) [inline pandas/core/ops.py]: this comment is not needed, as "sub" still belongs to the "Arithmetic operators" (the comment above)

**Review by jorisvandenbossche — COMMENTED** [review]
Git problems indeed fixed! 

Added two minor comments

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@kurtiskerstein Thanks for the PR!

**Pull request merged**